src/components/moderation/ModerationDetailsDialog.tsx

```diff
@@ -128,37 +130,82 @@ function ModerationDetailsDialogInner({
     description = ''
   }
 
+  const sourceName =
+    desc.source || desc.sourceDisplayName || _(msg`an unknown labeler`)
```

Comment:
Source will always be defined based on the logic in `useModerationCauseDescription`, but did this for completeness